docs/code-of-conduct-shortform-meetups.rst

```diff
@@ -1,18 +1,14 @@
-Code of Conduct
-===============
+Code of Conduct (Meetups)
+=========================
 
-Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation. Everybody who participates in our community in one way or another is required to conform to our `Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`_ (CoC). This includes online communications, meetup and conference attendees, speakers, sponsors, founders, moderators, organisers and volunteers.
+Write the Docs is dedicated to providing a safe, inclusive, welcoming, and harassment-free space and experience for all community participants, regardless of gender identity and expression, sexual orientation, disability, physical appearance, socioeconomic status, body size, ethnicity, nationality, level of experience, age, religion (or lack thereof), or other identity markers. 
```

Comment:
Is there a reason that this list of categories is different from https://www.writethedocs.org/code-of-conduct/ ?

The one we use there is:
> race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, religion, and mental and physical ability.

Reply:
These should probably be standardized.  Maybe even an include?